src/main/java/net/neoforged/neoforge/common/extensions/IItemStackExtension.java

```diff
@@ -507,4 +514,20 @@ default <T, C> T getCapability(ItemCapability<T, C> capability, C context) {
     default <T> T getCapability(ItemCapability<T, Void> capability) {
         return capability.getCapability(self(), null);
     }
+
+    /**
+     * {@return the attribute modifiers for the given equipment slot}
```

Comment:
Worth mentioning that this method is not idempotent and fires the `ItemAttributeModifierEvent` to compute the final modifiers.